sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -131,11 +129,13 @@ def __init__(self, **kwargs):
     self._group_names = []
     self.pipeline_labels = {}
     self.pipeline_annotations = {}
+    self.tekton_inline_spec = False
```

Comment:
@pugangxa Is it ok for us to compile custom task taskSpec as default? Right now we have to set a config flag to enable this feature, if your team can adopt this new behavior then we can make it as default.

Reply:
We discussed this today, and think as long as we can make sure all the ref cases(recursion) still works fine then it should be OK. From the code I saw we already excluded recursion so should work, please go ahead, thanks Tommy.